views/ancestorLines/ancestor_lines_explorer.js

```diff
@@ -2,6 +2,7 @@ import { AncestorTree } from "./ancestor_tree.js";
 import { showTree } from "./display.js";
 
 export class AncestorLinesExplorer {
+    static #COOKIE_NAME = "wt_ale_options";
```

Comment:
Just out of curiosity, why are you using hash (#) in front of your variable names? I've never seen that. Is that some kind of javascript convention?
In UML it's for protected fields, I think, but that's just pseudo / symbolic language, never seen that in actual code. 🤔

Reply:
Yes, in ES6 (at least, I think), it defines private variables

Reply:
Cool, thanks for confirmation. 👍